cajita/src/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ endif()
 
 if(Heffte_FOUND)
   target_link_libraries(Cajita Heffte::Heffte)
```

Comment:
```suggestion
  target_link_libraries(Cajita PUBLIC Heffte::Heffte)
```

Reply:
ooooh that might be it

Reply:
I've added the PUBLIC keyword to this and other instances of `target_link_libraries(Cajita...` but still seem to have the same issue.

Reply:
Okay adding the `PUBLIC` keyword and fixing a CMake error in CabanaMD looks like it fixes this. Cajita wasn't being properly linked there (just cabanacore)
So I think this now fixes #259 and is ready to merge.